bin/generate-feature-config.php

```diff
@@ -18,6 +18,14 @@
 $config_json = file_get_contents( 'config/' . $phase . '.json' );
 $config      = json_decode( $config_json );
 
+if ( ! empty( $_SERVER['WC_ADMIN_ADDITIONAL_FEATURES'] ) ) {
+	$additional_features = json_decode( $_SERVER['WC_ADMIN_ADDITIONAL_FEATURES'], true );
```

Comment:
This seems to work fine, but would `$_ENV` or `getenv()` be better than `$_SERVER` here? _(`$_SERVER` just makes me think we're in an HTTP request and not a cli script)_

Reply:
Updated to `$_ENV`. Agreed this makes more sense since they are environment variables. 

https://github.com/woocommerce/woocommerce-admin/pull/3044/commits/4e49066207713f134e7f52cecb966fbea3e757af.